docs/ref/templates/builtins.txt

```diff
@@ -515,15 +516,6 @@ of how ``x in y`` will be interpreted::
 
 Not contained within. This is the negation of the ``in`` operator.
 
-The comparison operators cannot be 'chained' like in Python or in mathematical
```

Comment:
I moved to the bottom of the comparison operators section, because it seemed odd to have it between `not in` and `is`. Do you think we should put it in a Note box to make it more visible?